SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1233,10 +1234,25 @@ void USpatialNetDriver::OnOwnerUpdated(AActor* Actor, AActor* OldOwner)
 
 void USpatialNetDriver::ProcessOwnershipChanges()
 {
+	const bool bShouldWriteLoadBalancingData =
+		IsValid(Connection) && GetDefault<USpatialGDKSettings>()->bEnableStrategyLoadBalancingComponents;
+	TOptional<const SpatialGDK::ActorSetWriter> LoadBalancingWriter;
+
+	if (bShouldWriteLoadBalancingData)
+	{
+		check(IsValid(PackageMap));
+		LoadBalancingWriter.Emplace(Connection->GetCoordinator(), *PackageMap);
+	}
+
 	for (Worker_EntityId EntityId : OwnershipChangedEntities)
 	{
 		if (USpatialActorChannel* Channel = GetActorChannelByEntityId(EntityId))
 		{
+			if (bShouldWriteLoadBalancingData)
+			{
+				LoadBalancingWriter->UpdateActorSetComponent(EntityId, Channel->Actor);
```

Comment:
Is there a reason this wasn't done entirely within ActorChannel::ServerProcessOwnershipChange?

Reply:
No real reason; I felt that this isn't really ActorChannel's job.

Reply:
I agree that perhaps it's not the channel's job, but as it's doing similar functionality there already I'd opt to keep it all situated in the same place. Not hugely opinionated though.